ui/txs/TxsListItem.tsx

```diff
@@ -31,24 +32,27 @@ type Props = {
   currentAddress?: string;
   enableTimeIncrement?: boolean;
   isLoading?: boolean;
+  translate?: boolean;
 }
 
 const TAG_WIDTH = 48;
 const ARROW_WIDTH = 24;
 
-const TxsListItem = ({ tx, isLoading, showBlockInfo, currentAddress, enableTimeIncrement }: Props) => {
+const TxsListItem = ({ tx, isLoading, showBlockInfo, currentAddress, enableTimeIncrement, translate }: Props) => {
   const dataTo = tx.to ? tx.to : tx.created_contract;
 
   const isOut = Boolean(currentAddress && currentAddress === tx.from.hash);
   const isIn = Boolean(currentAddress && currentAddress === tx.to?.hash);
 
   const timeAgo = useTimeAgoIncrement(tx.timestamp, enableTimeIncrement);
 
+  const { data: describeData } = NovesUseFetchDescribe(translate ? tx.hash : null);
```

Comment:
This functionality was discussed with the rest of the Blockscout team prior to submitting the PR, and it was agreed that it would be included in this page. The design mockups include it too. 

All of our APIs are prepared to handle this level of concurrency (we have layers of caching, etc in place). This calls a separate endpoint than the one called by the transaction detail page (`describeTx` instead of `tx`) because the `describe` endpoint has been optimized to be hit in a highly concurrent way.

So on this, we're going to include the functionality as originally agreed upon.

Reply:
I should also note that we designed this so that making the calls in parallel wouldn't affect first-time loading of the page. The UI will update as it gets the transaction types from the Noves API, but this shouldn't affect performance for loading the main view.

Reply:
Today, Nikita and I discussed this topic. We believe that a batch request is the most sensible solution. Our suggestion is to cover the tx status label with a skeleton until the data is loaded. If the request fails, we should display a regular tx type.

Reply:
Alright, we can set up a POST endpoint on our side that receives a batch of transaction hashes and returns the descriptions for all of them at once. Then the number of requests made from the browser is significantly reduced. Let's say we do this for 10 txs at a time? Thoughts?

Also, what would be the skeleton while the data is loaded? In the current version it displays the regular tx type and then replaces it with the Noves type when it arrives.

Reply:
Our standard page size is 50, so you can send all 50 transaction hashes together.

Regarding the skeleton, we suggest rendering the list item with all transaction information except for the transaction type, which will be covered with the skeleton. To achieve this, you can pass the `isLoading` prop to the `TxType` component, which will handle this state.